component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/util/TOTPUtil.java

```diff
@@ -532,11 +533,29 @@ public static boolean isEnrolUserInAuthenticationFlowEnabled(AuthenticationConte
                 TOTPAuthenticatorConstants.SUPER_TENANT_DOMAIN.equals(tenantDomain))) {
             return Boolean.parseBoolean(IdentityHelperUtil.getAuthenticatorParameters(
                     context.getProperty(TOTPAuthenticatorConstants.AUTHENTICATION).toString())
-                    .get(TOTPAuthenticatorConstants.ENROL_USER_IN_AUTHENTICATIONFLOW));
+                    .get(ENROL_USER_IN_AUTHENTICATIONFLOW));
         } else {
-            return Boolean.parseBoolean((context.getProperty(
-                    TOTPAuthenticatorConstants.ENROL_USER_IN_AUTHENTICATIONFLOW).toString()));
+            return Boolean.parseBoolean((context.getProperty(ENROL_USER_IN_AUTHENTICATIONFLOW).toString()));
+        }
+    }
+
+    /**
+     * Get EnrolUserInAuthenticationFlow.
+     *
+     * @return true, if EnrolUserInAuthenticationFlow is enabled
+     */
+    public static boolean isEnrolUserInAuthenticationFlowEnabled(AuthenticationContext context,
+                                                                 Map<String, String> runtimeParams) {
+
+        if (log.isDebugEnabled()) {
```

Comment:
`LOG.debug` method checked that check whether the debug is enabled. Since there is no string concatenation having the same check before adding the log is redundant